boards/atmel/sam/sam_v71_xult/Kconfig.sam_v71_xult

```
@@ -0,0 +1,11 @@
+# Copyright (c) 2016 Piotr Mienkowski
+# Copyright (c) 2024 Gerson Fernando Budke <nandojve@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_SAM_V71_XULT_SAMV71Q21
+	bool
+	select SOC_SAMV71Q21
+
+config BOARD_SAM_V71_XULT_SAMV71Q21B
+	bool
+	select SOC_SAMV71Q21B
```

Please, no bool, as the `BOARD_<name>` and `BOARD_<full_identifier>` are constructed here:
https://github.com/zephyrproject-rtos/zephyr/blob/db742d03aa8fa469a96435fe85b292e62d061d38/boards/Kconfig.v2#L8
https://github.com/zephyrproject-rtos/zephyr/blob/db742d03aa8fa469a96435fe85b292e62d061d38/boards/Kconfig.v2#L13

Some details can be found here: https://docs.zephyrproject.org/latest/build/kconfig/tips.html#common-kconfig-shorthands

Please do:
```suggestion
config BOARD_SAM_V71_XULT
	select SOC_SAMV71Q21 if BOARD_SAM_V71_XULT_SAMV71Q21
	select SOC_SAMV71Q21B if BOARD_SAM_V71_XULT_SAMV71Q21B
```
as that will ensure any typos or similar wrong selection is caughtr by CI.